firmware/Src/synth/Voice.cpp

```diff
@@ -5218,18 +5604,17 @@ void Voice::fxAfterBlock() {
             int drive = clamp(27 + sqrt3(fxParam1) * 86, 0, 255);
             float gain = 1.1f + 44 * panTable[drive];
             float gainCorrection = (1.2f - sqrt3(panTable[64 + (drive >> 1)] * 0.6f));
-            float bias = -0.1f + (fxParam1 * 0.2f);
 
             for (int k = BLOCK_SIZE; k--;) {
                 // Left voice
-                localv0L = tanh3(bias + sat33(*sp) * gain) * gainCorrection;
+                localv0L = tanh3(sat33(*sp) * gain) * gainCorrection;

```

Comment:
remove possible click when using that filter : bias not needed, that's not an analog filter